Misc/NEWS.d/next/Library/2022-04-15-17-38-55.gh-issue-91577.Ah7cLL.rst

```diff
@@ -0,0 +1 @@
+SharedMemory move imports out of methods
```

Comment:
```suggestion
Move imports in :class:`~multiprocessing.SharedMemory` methods to module level so that they can be executed late in python finalization. 
```

Reply:
Thanks, rebased and force pushed with the suggested change.